packages/transformers/posthtml/src/PostHTMLTransformer.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import {Transformer} from '@parcel/plugin';
+
+import posthtml from 'posthtml';
+import parse from 'posthtml-parser';
+import render from 'posthtml-render';
+import nullthrows from 'nullthrows';
+import semver from 'semver';
+import loadPlugins from './loadPlugins';
+
+const getPostHTMLConfig = async asset => {
```

Comment:
any reason why this function isn't inlined below?

Reply:
I had considered putting it in a `config.js`. I agree input would be better.

Reply:
Consolidated this.